frontends/web/src/routes/account/info/buyReceiveCTA.tsx

```diff
@@ -62,12 +63,23 @@ export const BuyReceiveCTA = ({ code, unit, balanceList, exchangeBuySupported =
 
 
 export const AddBuyReceiveOnEmptyBalances = ({ balances, accounts }: TAddBuyReceiveOnEmpyBalancesProps) => {
-
+  const mounted = useMountedRef();
   const [supportedAccounts, setSupportedAccounts] = useState<IAccount[]>();
 
   useEffect(() => {
-    getExchangeSupportedAccounts(accounts).then(setSupportedAccounts);
-  }, [accounts]);
+    if (mounted.current) {
+      getExchangeSupportedAccounts(accounts)
+        .then(supportedAccounts => {
+          if (mounted.current) {
+            setSupportedAccounts(supportedAccounts);
+          }
+        })
+        .catch(err => {
+          console.log('mounted.current', mounted.current);
+          console.error(err);
+        });
```

Comment:
this errors sometimes, see also commit message.

to reproduce: navigate to account, go back to account summary, unplug device.